packages/lab/src/avatar/Avatar.css

```diff
@@ -24,6 +27,7 @@
   overflow: hidden;
   user-select: none;
   border-radius: 50%;
+  outline: auto;
 }
 
 /* Style applied to the image/svg element. We specifically allow SVGs as the Avatar image to enable SVGs such as Flags to take up the full Avatar space */
```

Comment:
@el-dav @joshwooding do you know what are lines 38 to 41 for? i'm not sure what are they meant to do, hide text in svgs?

Reply:
I think it's the hide the broken image icon. It really only needs to be applied to img but worth checking